src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.js

```diff
@@ -57,7 +57,7 @@ const ImageRenderer = (props) => {
                 >
                     <ThumbnailImage
                         previewSourceURL={previewSource}
-                        style={styles.webViewStyles.tagStyles.img}
+                        style={styles.webViewStyles(1).tagStyles.img}
```

Comment:
`PixelRatio` should affect the image also right?

Reply:
No, images style doesn't have fontsize or lineHeight,

Reply:
If the img style doesn't need to know about the pixel ratio then we shouldn't pass an arbitrary variable here.